### PR TITLE
Embed RunWipeHistory into Modpack QoL Menu

### PR DESCRIPTION
**Pull request by Zyruvias**
- Add Cgull's RunWipeHistory Mod, accessible from the QoL Menu in the Modpack
- Change RunWipeHistory access from the "Assist" Button to QoL Menu to prevent text from showing up mid-run,

### CODE REVIEW

**Comment by cgu11** [comment]
we should probably redo this as a git submodule the same way we have many of the others, given that RunHistoryWipe has its own repo.

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,4 @@
 ################################################################################
 
 /.vs
+ModUtil/**
```
Comment (cgu11) [inline .gitignore:6]: why was this needed? not an issue more just curious

Comment (Zyruvias) [reply .gitignore:6]: It's not needed, just an artifact of me floundering around before I learned what submodules on GitHub actually are and how they worked.

> we should probably redo this as a git submodule the same way we have many of the others, given that RunHistoryWipe has its own repo.

Sounds good, I will work on getting that set up.

**Comment by Zyruvias** [comment]
@cgu11 This should be ready to go now, I have a pull request up on your repo and the submodule is set up. Let me know if you have any other feedback.

**Comment by Museus** [comment]
Cherry-picked this into `main-v130` to save us all the headache of rebasing it
